### ISSUE TITLE
Infinite warning shown...

### ISSUE THREAD

**Issue by malpower** [mqlight/nodejs-mqlight#16]
i created a example program trying to receive message from message hub, and it keep showing the following warning:
```
TimeoutOverflowWarning: Infinity does not fit into a 32-bit signed integer.
Timeout duration was set to 1.
```

the code is below:
```
const MqLight=require("mqlight");
const client=MqLight.createClient({service: "https://mqlight-lookup-prod02.messagehub.services.eu-gb.bluemix.net/Lookup?serviceId=xxxxxxx",
    user: "xxxxx",
    password: "xxxxxx"});
client.on("started",()=>
{
    console.log("Started");
    client.on("message",(msg)=>
    {
        console.log(msg);
    });
    client.subscribe("ttt",{autoConfirm: true});
});
client.on("error",(e)=>
{
    console.log(e);
});


const client2=MqLight.createClient({service: "https://mqlight-lookup-prod02.messagehub.services.eu-gb.bluemix.net/Lookup?serviceId=xxxxx",
user: "xxxxx",
password: "xxxxxx"});

client2.on("started", ()=>
{
    client2.send("ttt", "sdfsdfsdfdsfsdfsfdsf World!",{ttl: 1000});
});

client2.on("error",(e)=>
{
    console.log(e);
    process.exit(1);
});

```

BTW, the node version is v9.1.0

**Comment by malpower** [comment]
Any advise?

**Comment by Sammons** [comment]
@malpower this is a major issue our team also sees with Node 10+. It stems from this line: https://github.com/mqlight/nodejs-mqlight/blob/master/mqlight.js#L51

The change in behavior manifests itself in Node 9 (just look at the timers section)
https://nodejs.org/en/blog/release/v9.0.0/

Solution is to stop passing Infinity.

**Comment by Sammons** [comment]
@mimaison any chance this will change soon?

**Comment by goofy2b** [comment]
@mimaison, I also left a comment on the PR.  Do you have an ETA on when a new version will be published with the merged code?  Node 8 is going to be in maintenance mode later this year, and we would like to upgrade to Node 10.  The warning message seems to appear with Node 9 and Node 10. 
 With the warning messages flooding our logs, it is hold up on our Node 10 upgrade.

**Comment by SSAgovAdmin** [comment]
We want to upgrade one of our applications from Node 8 to Node 10, but this issue is preventing us from upgrading. The fix for this issue was merged in pull request #37. It appears that a new version of the library with the fix has not been published. Can someone provide a status of when a new version of the library with the fix will be published?

**Comment by dnwe** [comment]
@goofy2b / @SSAgovAdmin we've packaged up the most recent fixes as mqlight@2.0.2019041500 on [npm](https://npmjs.com/package/mqlight). Please can you test it out in your staging/development pipelines and confirm that the issue is resolved for you?

**Comment by SSAgovAdmin** [comment]
Thanks @dnwe ! That seems to have resolved the problem on our end. Thanks for the quick turnaround!
